src/assets/sass/protocol/includes/forms/_lib.scss

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Import all the things!
+@import '../lib';
+
+/* forms
```

Comment:
Should this file be in this PR or did it get copied over from the form work?

Reply:
Yup, forgot to delete it.

Reply:
Oh no, wait, I remember what I wanted it for.

Reply:
Updated it to just include the relevant variables for the buttons and removed those variables from the button file.